pkg/cmd/roachtest/tests/online_restore.go

```diff
@@ -242,6 +232,29 @@ func registerOnlineRestore(r registry.Registry) {
 	}
 }
 
+func postRestoreValidation(
+	ctx context.Context,
+	c cluster.Cluster,
+	l *logger.Logger,
+	dbName string,
+	approxDownloadJobEndTime time.Time,
+) error {
+	downloadJobHLC := hlc.Timestamp{WallTime: approxDownloadJobEndTime.Add(-time.Minute * 2).UnixNano()}
```

Comment:
Curious why we need to travel to the past? According to docs, [SHOW JOBS](https://www.cockroachlabs.com/docs/stable/show-jobs) reflects anything that ran in the past 12 hours.

Reply:
Added a comment to clarify. But the basic idea is that I want to verify that no automatic stats jobs are acting on the restoring tables during the download job. By running an AOST query at the download job end time, I can check that no job began before the download job finished. This query runs after the download job finishes, so without an AOST clause, I could catch an automatic job that began after the download job finished, which would lead to a false positive failure.

Reply:
Makes sense, thanks for clarifying. I guess I was thinking of why not just look at `created` timestamp, but AOST is way cooler :)